rag_experiment_accelerator/config/environment.py

```diff
@@ -92,6 +92,7 @@ class Environment:
     azure_document_intelligence_endpoint: Optional[str]
     azure_document_intelligence_admin_key: Optional[str]
     azure_key_vault_endpoint: Optional[str]
+    pypdf_enabled: Optional[bool]
 
```

Comment:
Severity: Potential Issue - Ensure that any new fields added to the Environment dataclass are also accounted for in any serialization or deserialization methods that operate on the class. This helps in maintaining consistent data handling and prevents errors.


[Hack - peer-review-pal] We are sorry for the intrusion. Feel free to ignore/delete the comments with the tag - they are automatically generated by [our Hack project](https://github.com/microsoft/peer-review-pal).
If you find them useful, want us to delete them or have any feedback or want to contribute - reach out to some of us: juliame@microsoft.com vadimkirilin@microsoft.com oteslenko@microsoft.com vimeena@microsoft.com
You can prevent future comments from appearing by adding #prp-ignore to your PR description.